src/scratchpad.js

```diff
@@ -132,7 +132,26 @@ export default function Scratchpad (DOM, props) {
     }
   };
 
+  const clientWidth$ = DOM.observable.pluck('clientWidth');
+
+  const mouseDowns$ = DOM.select('.handler').events('mousedown');
+  const mouseUps$ = DOM.select('.tricycle').events('mouseup');
+  const mouseMoves$ = DOM.select('.tricycle').events('mousemove');
+  const mouseLeaves$ = DOM.select('.tricycle').events('mouseleave');
+
+  const windowSizes$ = mouseDowns$
+    .flatMap(mouseDown => mouseMoves$.takeUntil(mouseUps$.merge(mouseLeaves$)))
+    .combineLatest(clientWidth$.throttle(100))
+    .map(([mouseDrag, clientWidth]) =>
+      (clientWidth - mouseDrag.clientX) / clientWidth
+    )
+    .filter(fraction => fraction < 0.9 && fraction > 0.1)
```

Comment:
Can we put these values into constants, declared at the top?

There are two major benefits. Constants improve readability by giving a name to the value, and they solve a problem where without them values get copied around the codebase and then it's hard to update them later (how do you know that every 0.9 is for the same thing)